WooCommerce/Classes/ViewRelated/Products/Edit Product/Product Settings/ProductSettingsRows.swift

```diff
@@ -154,6 +154,37 @@ enum ProductSettingsRows {
         let cellTypes: [UITableViewCell.Type] = [SettingTitleAndValueTableViewCell.self]
     }
 
+    struct EnableReviews: ProductSettingsRowMediator {
```

Comment:
I wonder if we should start writing unit tests for these `ProductSettingsRowMediator` instances. We can _probably_ test this in isolation. For example, we can test that the `ProductSettings` is updated whenever the `SwitchTableViewCell` _emits_ a change. 

Non-compiling pseudocode:

```swift
    func testWhenCellIsChangedThenReviewsAllowedIsUpdated() {
        // Given
        let settings = ProductSettings()
        let enableReviews = EnableReviews(settings: settings)

        let cell = SwitchTableViewCell()
        enableReviews.configure(cell: cell)

        // When
        cell.onChange(false)

        // Then
        XCTAssertEqual(settings.reviewsAllowed, false)
    }
```

Reply:
It looks good to me; I opened an issue here https://github.com/woocommerce/woocommerce-ios/issues/2526 to remember to add all the tests needed for `ProductSettingsRowMediator` 🎸